misc/ipc.h

```diff
@@ -0,0 +1,89 @@
+#ifndef IPC_H
+#define IPC_H
```

Comment:
As per https://github.com/greenbone/openvas-scanner/pull/1180 the new format is MISC_IPC_H. Same for all other header files.